Process.c

```diff
@@ -739,9 +739,6 @@ void Process_printPercentage(float val, char* buffer, int n, int* attr) {
             *attr = CRT_colors[PROCESS_SHADOW];
          }
          xSnprintf(buffer, n, "%4.1f ", val);
-      } else if (val < 999) {
-         *attr = CRT_colors[PROCESS_MEGABYTES];
-         xSnprintf(buffer, n, "%3d. ", (int)val);
       } else {
          *attr = CRT_colors[PROCESS_MEGABYTES];
          xSnprintf(buffer, n, "%4d ", (int)val);
```

Comment:
```suggestion
      } else {
         *attr = CRT_colors[PROCESS_MEGABYTES];
         if (val < 100.0F)
            val = 100.0F; // Don't round down or display as "99".
         xSnprintf(buffer, n, "%4.0f ", val);
```

Reply:
@Explorer09, Thank you! for the suggestion, but I couldn't figure out why do we need following if statement.

```c
if (val < 100.0F)
    val = 100.0F; // Don't round down or display as "99".
```

As we're using `%4.0f` for formatting, it would automatically round all value between `(99.49, 100.49]` to 100. And `if (val < 99.9F)` statement make sure our `val` always greater than `99.9`, so I don't think we above if statement, at all.

Reply:
Problem is that `printf` respects the rounding mode, which could be set to always round down. Under that circumstances `printf("%4.0f", 99.9);` would actually print `  99`, not ` 100` as you'd expect.